docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
```

Comment:
```suggestion
Data Compression
----------------
```

Reply:
Data compression

(use sentence case in titles..)

Reply:
AC